components/Auth/login.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- font awesome -->
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <!-- styles -->
+    <link rel="stylesheet" href="https://rapidui.netlify.app/styles/main.css" />
+    <link rel="stylesheet" href="../../index.css" />
+    <link rel="stylesheet" href="./auth.css" />
+
+    <title>Shop | Rapid Store</title>
+  </head>
+  <body>
+    <div class="sub-nav">
+      <div class="left">
+        <a href="#">About Us</a>
+        <a href="#">Privacy Policy</a>
+        <a href="#">Order Tracking</a>
+      </div>
+      <div class="right">
+        <span>Need help? Call Us: 000 000 0000</span>
+        <a href="#">Contact Us</a>
+        <a href="https://github.com/vleads"
+          ><i class="fab fa-github" aria-hidden="true"></i
+        ></a>
+        <a href="https://twitter.com/vishalk01234"
+          ><i class="fab fa-twitter" aria-hidden="true"></i
+        ></a>
+        <a href="https://www.linkedin.com/in/vishalkumar28/"
+          ><i class="fab fa-linkedin" aria-hidden="true"></i
+        ></a>
+        <a href="mailto:leader.vishalkumar@gmail.com"
+          ><i class="fas fa-envelope" aria-hidden="true"></i
+        ></a>
+      </div>
+    </div>
+
+    <nav class="navbar">
+      <a class="navbar-logo" href="../../index.html">
+        <img src="../../assets/img/rapidstore.png" alt="rapid store" />
+      </a>
+      <div class="search-box">
+        <i class="fas fa-search"></i>
+        <input type="text" placeholder="Search" />
+      </div>
+      <ul class="nav-links">
+        <li>
+          <a href="#" class="btn btn-primary">Login</a>
+        </li>
+        <li>
+          <a href="../Wishlist/wishlist.html"
```

Comment:
try to have folders' names with lowercase.

Reply:
Changed names to lowercase.